File: fetcher/app/process.js
var vdf = require('node-vdf');
var fs = require("fs");

var file = process.argv[2];
var branch = process.argv[3];
var field = process.argv[4];

fs.readFile(file, function (err, data) {
    if (err) throw err;

    var rawVDFWithHeader = data.toString();

    var vdfWithoutAppHeader = rawVDFWithHeader.split('"544550"', 2);

    var saneVDF = '"544550"\n' + vdfWithoutAppHeader[1];

    var parsedVDF = vdf.parse(saneVDF);

    console.log(parsedVDF["544550"]["depots"]["branches"][branch][field]);
});

File: fetcher/app/process.py
import vdf
import sys

fileName = sys.argv[1]
branch = sys.argv[2]
field = sys.argv[3]
steamid = sys.argv[4]

with open(fileName, 'r') as f:
        rawFile = f.read()
        d = "\"" + steamid + "\""
        removedTrashBeginning = [d+e for e in rawFile.split(d, 1) if e]
        
        revTrash = removedTrashBeginning[1][::-1]

        d = "}"
        removedTrashEnd = [d+e for e in revTrash.split(d, 1) if e]

        cleanedFile = removedTrashEnd[1][::-1]

        data = vdf.loads(cleanedFile)

        print(data[steamid]["depots"]["branches"][branch][field])